indico/modules/events/editing/controllers/backend/management.py

```diff
@@ -87,3 +96,43 @@ def _process_DELETE(self):
                 raise ExpectedError(_('Cannot delete the only publishable file type'))
         delete_file_type(self.file_type)
         return '', 204
+
+
+class RHEditingReviewConditions(RHEditingManagementBase):
+    def _process_GET(self):
+        return jsonify(editing_settings.get(self.event, 'review_conditions').items())
+
+    @use_rh_kwargs(EditingReviewConditionArgs)
+    def _process_POST(self, file_types):
+        new_conditions = OrderedDict(editing_settings.get(self.event, 'review_conditions'))
+        new_conditions[unicode(uuid4())] = file_types
+        editing_settings.set(self.event, 'review_conditions', new_conditions)
+        return '', 204
+
+
+class RHEditingEditReviewCondition(RHEditingManagementBase):
+    def _process_args(self):
+        RHEditingManagementBase._process_args(self)
+        uuid = request.view_args['uuid']
+        try:
+            UUID(uuid)
```

Comment:
tip, you can use `<uuid:uuid>` in the url rule, then it'll only match on  valid uuids

Reply:
Ah, true.. I forgot about it.. (we could change other places that don't use it)